### ISSUE TITLE
Projects not loading - TRPC returns 400

### ISSUE THREAD

**Issue by samajammin** [gitcoinco/easy-retro-pgf#13]
Noticing that occasionally when routing from /ballot to /projects, the projects don't load

e.g. navigating to projects via the nav link: 
https://easy-retro-pgf.vercel.app/projects?orderBy=name&sortOrder=asc&limit=12&cursor=0&seed=0&search=null

![Image 2024-01-29 at 13 36 37](https://github.com/gitcoinco/easy-retro-pgf/assets/8097623/0a17ac44-3827-4923-932e-fb397dc5179b)

Looks like this request is failing:
https://easy-retro-pgf.vercel.app/api/trpc/projects.search?batch=1&input=%7B%220%22%3A%7B%22json%22%3A%7B%22limit%22%3A%2212%22%2C%22seed%22%3A%220%22%2C%22orderBy%22%3A%22name%22%2C%22sortOrder%22%3A%22asc%22%2C%22cursor%22%3Anull%2C%22search%22%3A%22null%22%7D%2C%22meta%22%3A%7B%22values%22%3A%7B%22cursor%22%3A%5B%22undefined%22%5D%7D%7D%7D%7D

```
[
{
"error": {
"json": {
"message": "[\n  {\n    \"code\": \"invalid_type\",\n    \"expected\": \"number\",\n    \"received\": \"string\",\n    \"path\": [\n      \"limit\"\n    ],\n    \"message\": \"Expected number, received string\"\n  },\n  {\n    \"code\": \"invalid_type\",\n    \"expected\": \"number\",\n    \"received\": \"string\",\n    \"path\": [\n      \"seed\"\n    ],\n    \"message\": \"Expected number, received string\"\n  }\n]",
"code": -32600,
"data": {
"code": "BAD_REQUEST",
"httpStatus": 400,
"path": "projects.search",
"zodError": {
"formErrors": [],
"fieldErrors": {
"limit": [
"Expected number, received string"
],
"seed": [
"Expected number, received string"
]
}
}
}
}
}
}
]
```

**Comment by samajammin** [comment]
When I remove the query params & hard refresh at https://easy-retro-pgf.vercel.app/projects it seems to work

**Comment by carlbarrdahl** [comment]
Thank you @samajammin!
Should be fixed in fb4e97177b3e311881c830b769f3206ffa9b417c
